packages/resolver/lib/sources/truffle/index.ts

```diff
@@ -68,6 +73,25 @@ export class Truffle implements ResolverSource {
         this.options.compilers
       );
       return { body: addressSource, filePath: importPath };
+    } else if (importPath === `truffle${path.sep}Console.sol`) {
```

Comment:
I already :heavy_check_mark: the PR, but I do think this should also accept `truffle${path.sep}console.sol`

Reply:
`Console.sol` is more consistent with the other truffle sources. What's your thinking?

Reply:
While the consistency is nice, the other truffle sources are only to be used in Solidity-based tests. This is the first contract of its kind, and it aligns with HH's, Ganache's, and maybe most importantly, the name of the contract library itself.

> Contract and library names should also match their filenames.

- https://docs.soliditylang.org/en/v0.8.17/style-guide.html#contract-and-library-names

The library name is `console`. So I think the file should be `console.sol` to match.

Reply:
cc @gnidan

Reply:
Definitely prefer aligning with OZ's use of PascalCase for Solidity files. Unless they're just matching the exported symbol.

Seems like `library console` is bad style? For clear reasons, but still.

Reply:
Yeah, `library console`, is not idiomatic Solidity, but `console.log` isn't either.

Since library names should match their filenames I think Truffle must either align its import filename with the library's name (`console`), or rewrite the library to match the proposed filename of `truffle/Console.sol` (so usage would be `Console.log` instead of `console.log` -- preventing drop in replacement).

Reply:
How about allowing both `@truffle/[Cc]ontract.sol`, promote?